### ISSUE TITLE
Unable to Launch on macOS Mojave 

### ISSUE THREAD

**Issue by DavidWiesner** [Lymphatus/caesium-image-compressor#33]
After installing the v2.0.0-alpha.2 an alert dialog will be open with the message

Caesium Image Compressor cannot be opened because of a problem.

 Check with the developer to make sure Caesium Image Compressor works with this version of macOS. You may need to reinstall the application. Be sure to install any available updates for the application and macOS.

```
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: libzopfli.1.dylib
  Referenced from: /Applications/Caesium Image Compressor.app/Contents/Frameworks/libcaesium.dylib
  Reason: image not found

Binary Images:
       0x10978e000 -        0x1097b7fff +com.saerasoft.caesium (0) <62F31A04-1B12-3B5F-8388-B08CB1992FC5> /Applications/Caesium Image Compressor.app/Contents/MacOS/Caesium Image Compressor
       0x1097d4000 -        0x109826ff7 +libcaesium.dylib (0) <01EBDDC5-B91C-3F46-8470-5716F0486057> /Applications/Caesium Image Compressor.app/Contents/Frameworks/libcaesium.dylib
       0x10982f000 -        0x109862ff3 +org.qt-project.QtSvg (5.12 - 5.12.8) <6312A82E-B9E0-3817-9D16-AA3E2134DC5C> /Applications/Caesium Image Compressor.app/Contents/Frameworks/QtSvg.framework/Versions/5/QtSvg
       0x109881000 -        0x109cbfff7 +org.qt-project.QtWidgets (5.12 - 5.12.8) <CAC50E2E-DC1C-37F4-8BEF-9953C603230F> /Applications/Caesium Image Compressor.app/Contents/Frameworks/QtWidgets.framework/Versions/5/QtWidgets
       0x109e28000 -        0x10a276fe7 +org.qt-project.QtGui (5.12 - 5.12.8) <6D5A30B0-E836-303B-B20F-BA809ABF7D57> /Applications/Caesium Image Compressor.app/Contents/Frameworks/QtGui.framework/Versions/5/QtGui
       0x10a37e000 -        0x10a382ff7 +org.qt-project.QtConcurrent (5.12 - 5.12.8) <A6F0EAB9-D122-3FDE-A112-280F346032C0> /Applications/Caesium Image Compressor.app/Contents/Frameworks/QtConcurrent.framework/Versions/5/QtConcurrent
       0x10a386000 -        0x10a8bbff7 +org.qt-project.QtCore (5.12 - 5.12.8) <AB81B515-9C42-3545-B1D2-06E8090C5422> /Applications/Caesium Image Compressor.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore
       0x10a97e000 -        0x10aa1dff3 +libjpeg.62.dylib (0) <F0BE6AD8-219D-333F-8AD1-8085C3399171> /Applications/Caesium Image Compressor.app/Contents/Frameworks/libjpeg.62.dylib
       0x10aa2a000 -        0x10aadeffb +libturbojpeg.0.dylib (0) <FC9DEF68-35D5-39FB-AF15-79D7E401D5EB> /Applications/Caesium Image Compressor.app/Contents/Frameworks/libturbojpeg.0.dylib
       0x114dfc000 -        0x114e8df47  dyld (750.6) <F9D4DEDC-8296-3E3F-B517-9C8B89A4C094> /usr/lib/dyld
    0x7fff293ce000 -     0x7fff293d2ffb  com.apple.agl (3.3.3 - AGL-3.3.3) <204E20A9-AEA9-3F9F-AE23-EECE3AFFFC54> /System/Library/Frameworks/AGL.framework/Versions/A/AGL
    0x7fff2feb0000 -     0x7fff2feb6fff  com.apple.DiskArbitration (2.7 - 2.7) <63923E3F-1489-3762-B5EB-5CE28A35FF50> /System/Library/Frameworks/DiskArbitration.framework/Versions/A/DiskArbitration
    0x7fff3092a000 -     0x7fff309ceff3  com.apple.framework.IOKit (2.0.2 - 1726.140.1) <14223387-6F81-3976-8605-4BC2F253A93E> /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
    0x7fff386b6000 -     0x7fff386c5ff7  com.apple.opengl (17.10.22 - 17.10.22) <D3C57A32-6BD0-3228-B1C4-0F42A6128A6C> /System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL
    0x7fff64c31000 -     0x7fff64c32fff  libSystem.B.dylib (1281.100.1) <C0C9872A-E730-37EA-954A-3CE087C15535> /usr/lib/libSystem.B.dylib
    0x7fff64f17000 -     0x7fff64f69fff  libc++.1.dylib (902.1) <59A8239F-C28A-3B59-B8FA-11340DC85EDC> /usr/lib/libc++.1.dylib

```

**Comment by DavidWiesner** [comment]
I have found a workaround for this. 
```bash
cd /Applications/Caesium Image Compressor.app/Contents/Frameworks
install_name_tool -change libzopfli.1.dylib @rpath/libzopfli.1.dylib libcaesium.dylib
install_name_tool -change libzopflipng.1.dylib @rpath/libzopflipng.1.dylib libcaesium.dylib
install_name_tool -change libzopfli.1.dylib @rpath/libzopfli.1.dylib libzopflipng.1.dylib
```

**Comment by Lymphatus** [comment]
Seems like the same problem with #25 but on different OS.

**Comment by Lymphatus** [comment]
Should be resolved with `v2.0.0-alpha.3`.
